app/graphql/types/submission_info_type.rb

```diff
@@ -28,10 +28,13 @@ def milestone_number
         .for(object.target_id)
         .batch do |target_ids, loader|
           Target
+            .includes(:assignments)
             .where(id: target_ids)
-            .each { |target| loader.call(target.id, target.milestone_number) }
+            .each do |target|
+              loader.call(target.id, target.assignments.first.milestone_number)
+            end
         end
-      # object.target.milestone_number
+      # object.target.assignments.first.milestone_number
```

Comment:
This is a comment... why update it? Let's clean up instead?

Reply:
@harigopal I thought the comment was trying to show in a single line what the batchloader is trying to do? There is a similar comment for every batch loader query.

Reply:
@shilendras It's probably left over from when these queries weren't batched. 😅

Reply:
Okay! 😄 Cleaned up the comments.